bsip-0086.md

```diff
@@ -44,42 +45,46 @@ the scheduled time, at the scheduled time and after the scheduled time.
 
 ## New global parameter
 
-Add a new global parameter `market_fee_network_percent` which can be updated
+Add a new global parameter `market_fee_committee_percent` which can be updated
 by the committee only after the protocol upgrade.
 Initial value of that parameter is `0%`.
 Valid range of that parameter is `[0%, 100%]`.
 
+UPDATE: limit `[0,100%]` to `[0,30%]` see: https://github.com/bitshares/bsips/pull/269
```

Comment:
The wording change to make it clear it goes to committee seems reasonable.

On the marked change here: https://github.com/bitshares/bsips/issues/194#issuecomment-607404101